frontend/src/container/MetricsApplication/Tabs/External/Table/Toolbar.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, Input, Radio, Space } from 'antd';
+import React from 'react';
+
+import { ReduceTo } from './reduceTo';
+
+export type FilterDropdownProps = {
+	addressFilter: string;
+	onAddressFilterChange: (v: string) => void;
+	reduceTo: ReduceTo;
+	onReduceToChange: (v: ReduceTo) => void;
+};
+
+function FilterDropdown(props: FilterDropdownProps): JSX.Element {
+	const {
+		addressFilter,
+		onAddressFilterChange,
+		reduceTo,
+		onReduceToChange,
+	} = props;
+
+	return (
+		<Card size="small">
+			<Space
+				align="start"
+				direction="horizontal"
+				style={{ width: '100%', justifyContent: 'space-between' }}
+			>
+				<Input
+					placeholder="Search by address"
+					value={addressFilter}
+					onChange={(e): void => onAddressFilterChange(e.target.value)}
```

Comment:
I fixed the other comments. Please check. 🙂

---

Not sure about this one.

Do you want the same for `<Radio.Group>`?

https://github.com/visortelle/signoz/blob/77ec1b14ba3fe39fb91509aa2abaf459a64816e7/frontend/src/container/MetricsApplication/Tabs/External/Table/Toolbar.tsx#L37

---

Just can't imagine the situation why the `<Toolbar />` parent component should care about events instead of a `string` value. 
Or in the case of `<Radio.Group>`, the AntD-specific event.

In the case of reusable components across the app or lib, I understand why.

I'll fix it if you want. Just curious.

Reply:
no will i think that situation is coming right now... would recommend at that moment only